templates/clients/clients_cl.html

```diff
@@ -1,4 +1,30 @@
 {{ define "page" }}
+
+{{ $root := . }}
+
+  <!-- Peer details modal -->
+  <div class="modal modal-xl" id="peerDetailsModal" tabindex="-1" role="dialog" aria-labelledby="peerDetailsModalTitle" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered modal-xl" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title" style="overflow-wrap: anywhere;">
+            <svg id="peerDetailsModalTitleImage" class="peer-table-icon" style="width:50px;height:50px;" data-jdenticon-value="user127"></svg>
+            <span id="peerDetailsModalTitle"></span>
+          </h5>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"  onclick="$('#peerDetailsModal').modal('hide')">
```

Comment:
```suggestion
          <button type="button" class="close btn btn-outline-secondary" data-dismiss="modal" aria-label="Close"  onclick="$('#peerDetailsModal').modal('hide')">
```
design fix

Reply:
Good eye! Instead of your suggestion, I went with `class="btn-close"`, looks a bit better.